finetuner/constants.py

```diff
@@ -0,0 +1,5 @@
+DELETE = "DELETE"
+POST = "POST"
+GET = "GET"
+NAME = "name"
+USER_ID = "user_id"
```

Comment:
is this necessary? btw, always use single `'` quotes, you can apply `black -S .`

Reply:
I think it's nice to use constants and have a separate file for them. I prefer keeping it if you don't have a strong opinion.